library/src/test/java/net/jimblackler/jsonschemafriend/SchemaStoreTest.java

```diff
@@ -180,8 +182,8 @@ private Collection<DynamicNode> test(String dirName, boolean mustFail) {
 
             System.out.println(objectWriter.writeValueAsString(output));
 
-            assertTrue(extraReported.isEmpty(), "Errors reported not seen in reference file");
-            assertTrue(notReported.isEmpty(), "Errors in reference file not reported");
+            assertThat("Errors reported not seen in reference file",  extraReported, is(empty()));
+            assertThat("Errors in reference file not reported", notReported, is(empty()));
```

Comment:
Switching to hamcrest prints out the content of these when not empty. Which is useful for tracking down errors that only happen on the build server.